en/docs/develop/instagram-authenticator.md

```diff
@@ -1,5 +1,10 @@
 # Configuring Instagram Authenticator
 
+!!! note
+        This authenticator is now deprecated as the Instagram API 
+	Platform which supported authentication is no longer available. 
+	The [Facebook Authenticator](https://is.docs.wso2.com/en/latest/learn/configuring-facebook/) can be used as an alternative to this.
```

Comment:
(https://is.docs.wso2.com/en/latest/learn/configuring-facebook/)

Please give the relative link: (../../learn/configuring-facebook)

Reply:
Added this in the following PR.
[docs-is/pull/1411](https://github.com/wso2/docs-is/pull/1411)